src/test/java/com/github/dockerjava/core/command/ExecStartCmdImplTest.java

```diff
@@ -92,4 +90,25 @@ public void execStartAttached() throws Exception {
         assertNotNull(responseAsString);
         assertTrue(responseAsString.length() > 0);
     }
+    
+    @Test(groups = "ignoreInCircleCi", expectedExceptions = NotFoundException.class)
```

Comment:
Ok so it's the below line that throws the NFE. My thinking was to remove everything below this line and keep the `expectedExceptions = NotFoundException.class`. This shows that creating with some non-existent-user disallows one to copy anything from the container, in this case `/execStartTest.log`, because it does not exist because the user itself does not exist.

`InputStream response = dockerClient.copyArchiveFromContainerCmd(container.getId(), "/execStartTest.log").exec();`